validators_instructions.md

```diff
@@ -0,0 +1,139 @@
+
+It can run in single signer mode or as a voter in the Cosmos sommelier protocol.
+
+It integrates the full functionality of gorc for operating as an orchestrator and relayer of gravity bridge messages between the Ethereum and Cosmos chains.
+
+## Background
+
+Every automated investing system has a feedback loop where changes in the market must be observed and investments rebalanced in response. 
+
+Steward closes the loop in the Sommelier chain. For instance, it allows Validators to vote for rebalancing Cellar funds in response to a Data Provider’s recommendation based on market changes. 
+
+Additionally, Steward houses the Orchestrator, which is responsible for interactions with both chains via the Gravity Bridge.
+
+### **Cellars**
+
+Cellars are smart contracts that are designed to manage funds based on a particular investment strategy.
+
+### **Data Providers**
+
+Data Providers (DPs) are users or organizations who provide recommended rebalance arguments based on price or other market data to the Sommelier validators in exchange for a fee. There can be multiple Data Providers per strategy, and Validators are responsible for deciding which one they pay in exchange for data.
+
+### **Sommelier Chain**
+
+The Sommelier chain is a Cosmos SDK application chain. It is the core of Sommelier that makes decisions about whether to act on DP recommendations and executes Cellar functions. Transactions and function calls are only settled on-chain when the Validators reach consensus. In order to fulfill a strategy, Validators need a way to receive recommendation data from the Data Providers, reach consensus on whether to act on that data, and then call functions on the Cellars with the provided data as parameters. This is where Steward comes in.
+
+### **Steward**
+
+Steward runs on every Validator in the Sommelier Validator set. It runs a server to which Data Providers send recommendations as soon as market data becomes available. This payload contains identifying information about the Cellar which Steward is meant to act on, and the recommended arguments for its rebalance function parameters. When Steward receives a new recommendation, it submits a vote to rebalance the appropriate Cellar based on that data. If Validators reach a quorum, the process of rebalancing begins.
+
+Since Sommelier is a Cosmos chain, in order to provide strategies on Ethereum it needs a way to bridge assets and logic calls. Steward is responsible for running the Orchestrator, which handles relaying Cosmos transactions to Ethereum, and co-processing Ethereum transactions on Sommelier. Steward runs the Orchestrator so that Sommelier can manage Cellars on Ethereum.
+
+## Setting Up Steward
+
+In this section, let’s explore setting up steward for validators. First, ensure the sommelier chain is running. Next, create a `toml` file in the root of the application which will hold your configuration. To get a template for your configuration file, run the command below:
+
+```rust
```

Comment:
The configs for the triple backtick stuff in Markdown are meant to render the text based on the syntax. You probably want `bash` here (and elsewhere throughout the readme files) since these are commands for the CLI.

Reply:
fixed